test/util/namespaces/namespaces.go

```diff
@@ -112,7 +112,25 @@ func CleanNetworks(operatorNamespace string, cs *testclient.ClientSet) error {
 			}
 		}
 	}
-	return err
+	return waitForSriovNetworkDeletion(operatorNamespace, cs, 15*time.Second)
+}
+
+func waitForSriovNetworkDeletion(operatorNamespace string, cs *testclient.ClientSet, timeout time.Duration) error {
+	return wait.PollImmediate(time.Second, timeout, func() (bool, error) {
+		networks := sriovv1.SriovNetworkList{}
+		err := cs.List(context.Background(),
+			&networks,
+			runtimeclient.InNamespace(operatorNamespace))
+		if err != nil {
```

Comment:
Can you return true if errors.IsNotFound? I am not sure how List behaves if there are no entries.

Reply:
List() just returns an empty .Items list if not results are found

Reply:
Good to know!